### PR TITLE
SMTP relay integration

### PR DESCRIPTION
**Pull request by jeyrey**

### CODE REVIEW

File: config/production.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
 	"db_name" : "sqlite3",
 	"db_path" : "gophish.db",
 	"migrations_prefix" : "db/db_",
-	"environment": "development"
+	"environment": "development",
+	"block_test_email": true
```
[the remark this answers is not in the document]
Comment (jeyrey) [reply config/production.json]: You should make it false if you want to send 3 emails every time you delete gophish.db and run it again.

File: gophish.go [hunk 2]
```diff
@@ -26,34 +26,53 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
...
 	"github.com/teamnsrg/gophish/models"
 	"github.com/teamnsrg/gophish/util"
-	"github.com/gorilla/handlers"
 )
 
 var (
 	Logger = log.New(os.Stdout, " ", log.Ldate|log.Ltime|log.Lshortfile)
 
-	configPath = kingpin.Flag("config", "Location of config json file").Default("./config/development.json").String()
+	configPath = kingpin.Flag("config", "Location of config json file").Default(os.Getenv("GOPATH") + "/src/github.com/teamnsrg/gophish/config/development.json").String()
```
[the remark this answers is not in the document]
Comment (jeyrey) [reply gophish.go]: I was trying to get gophish to run even if it is executed from another directory. Its a step in that direction

File: run [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "Starting Gophish..."
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root. Don't forget that GOPATH may be different for root. <sudo -E ./run>"
+  exit
+fi
+
+source $GOPATH/src/github.com/teamnsrg/gophish/config/tmail.cfg
+/mnt/c/School/work/repos/all-things-go/src/github.com/teamnsrg/gophish/gophish
```
[the remark this answers is not in the document]
Comment (jeyrey) [reply run]: Everyone should use Windows :)

File: config/production.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (zzma) [reply config/production.json]: Yes - but this is the production config, where we won't be using gophish.db and we do want to send emails. This flag should really be `block_email` and block all email sending when true, and be set to false for production.

File: gophish.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (zzma) [reply gophish.go]: The flip side is that gophish assumes the user has $GOPATH set and can't just run the binary without installing Go. I'm fine with it, and was just curious

File: config/production.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeyrey) [reply config/production.json]: I can make it do that

File: ssl/web_server.key [hunk 4]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
[the remark this answers is not in the document]
Comment (jeyrey) [reply ssl/web_server.key]: This is just the TLS stuff for the SMTP relay. It only connects over the loopback interface anyway so we really dont need TLS. I just added in placeholder keys so we could make minimal changes to the tmail project code.

**Pull request merged**